app/models/issue.rb

```diff
@@ -49,4 +51,9 @@ def sub_id
   def self.find_from_project(project, sub_id)
     Issue.find(project.issue_ids[sub_id.to_i - 1])
   end
+
+  def set_sub_id
+    current_id = project.issues.count
```

Comment:
the sub_id always starts from 1, because it's only set to the number of issues in a project after an issue is created (sub_id = project.issues.count)
![2](https://cloud.githubusercontent.com/assets/6087164/6733991/3bb0a664-ce5e-11e4-9da1-30da911ee015.png)